continuous_integration/travis/install.sh

```diff
@@ -1,54 +0,0 @@
-set -xe
-
-#!/usr/bin/env bash
-# Install conda
-case "$(uname -s)" in
-    'Darwin')
-        MINICONDA_FILENAME="Miniconda3-latest-MacOSX-x86_64.sh"
-        ;;
-    'Linux')
-        MINICONDA_FILENAME="Miniconda3-latest-Linux-x86_64.sh"
-        ;;
-    *)  ;;
-esac
-
-
-# Install miniconda
-wget https://repo.continuum.io/miniconda/$MINICONDA_FILENAME -O miniconda.sh
-bash miniconda.sh -b -p $HOME/miniconda
-export PATH="$HOME/miniconda/bin:$PATH"
-export BOTO_CONFIG=/dev/null
-conda config --set always_yes yes --set changeps1 no --set remote_max_retries 10
-
-# Create conda environment
-conda env create -q -n test-environment -f $ENV_FILE
-source activate test-environment
-
-# We don't have a conda-forge package for cityhash
-# We don't include it in the conda environment.yaml, since that may
-# make things harder for contributors that don't have a C++ compiler
-python -m pip install --no-deps cityhash
```

Comment:
Looks like we are no longer installing `cityhash` in the new `install.sh` script